app/config.py

```diff
@@ -244,6 +242,9 @@ class Live(Config):
     FROM_NUMBER = '40604'
     FUNCTIONAL_TEST_PROVIDER_SERVICE_ID = '6c1d81bb-dae2-4ee9-80b0-89a4aae9f649'
     FUNCTIONAL_TEST_PROVIDER_SMS_TEMPLATE_ID = 'ba9e1789-a804-40b8-871f-cc60d4c1286f'
+    PERFORMANCE_PLATFORM_ENABLED = True
+    PERFORMANCE_PLATFORM_URL = 'https://www.performance.service.gov.uk/data/govuk-notify/notifications'
+    PERFORMANCE_PLATFORM_TOKEN = os.getenv('PERFORMANCE_PLATFORM_TOKEN')
```

Comment:
this won't work on its own - you need to add the relevant line to the `cloudfoundry_config.py` file